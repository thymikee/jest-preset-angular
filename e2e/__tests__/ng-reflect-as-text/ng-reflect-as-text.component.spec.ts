import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { NgReflectAsTextComponent } from './ng-reflect-as-text.component';

describe('NgReflectAsTextComponent', () => {
  let fixture: ComponentFixture<NgReflectAsTextComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [NgReflectAsTextComponent],
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(NgReflectAsTextComponent);
    fixture.detectChanges();
  });

  test('snapshots', () => {
    expect(fixture).toMatchSnapshot();
  });
});
