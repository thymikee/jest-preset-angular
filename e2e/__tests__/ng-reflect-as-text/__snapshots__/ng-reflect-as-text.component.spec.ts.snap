// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NgReflectAsTextComponent snapshots 1`] = `
<app-ng-reflect-as-text
  __ngContext__={[Function LRootView]}
>
  <p
    data-some-attr="ng-reflect-as-attr-arg works"
  >
     ng-reflect-as-text works!
  
  </p>
</app-ng-reflect-as-text>
`;
